example_contracts/libraries/using_for_simple.sol

```diff
@@ -21,3 +13,12 @@ contract WARP {
     return t.amIReal(d);
   }
 }
+
+library Nums {
```

Comment:
What's the reasoning behind moving this?

Reply:
Was using `visitContractDefinition` prior for a part to accomplish it. It was a check for library declared after contract. Can we keep it this was for behaviour testing?

Reply:
I'd suggest having one behaviour test with it declared before, one with it declared after, and if it works or is easily fixable, one with it being imported from an adjacent file. Might as well cover all bases, especially as the sourceUnitSplitter is liable to change in the future